media/js/glean/init.es6.js

```diff
@@ -5,22 +5,23 @@
  */
 
 import Glean from '@mozilla/glean/web';
+import GleanMetrics from '@mozilla/glean/metrics';
 import { BrowserSendBeaconUploader } from '@mozilla/glean/web';
-import { initPageView, pageEvent } from './page.es6';
+import { recordCustomPageMetrics, pageEvent } from './page.es6';
 import { clickEvent } from './elements.es6';
 import Utils from './utils.es6';
 
-const shouldInitialize = Utils.isValidHttpUrl(window.location.href);
-
 function initGlean() {
     const pageUrl = window.location.href;
     const endpoint = 'https://www.mozilla.org';
     const channel = pageUrl.startsWith('https://www.mozilla.org/')
```

Comment:
Nit(non-blocking): You could replace the string on this line with the `endpoint` var since you just declared it.

Reply:
Good suggestion, thanks!